pkgs/by-name/op/openthread-border-router/package.nix

```diff
@@ -0,0 +1,129 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  systemdLibs,
+  avahi,
+  protobuf,
+  jsoncpp,
+  boost,
+  libnetfilter_queue,
+  libnfnetlink,
+  nodejs,
+  buildNpmPackage,
+}:
+let
+  pname = "ot-br-posix";
+  version = "unstable-2024-10-18";
+
+  src = fetchFromGitHub {
+    owner = "openthread";
+    repo = "ot-br-posix";
+    rev = "76bfeef4e3e0242c235aa6bdf5480a0642883599";
+    hash = "sha256-VtekJD++PAse+yNmWzpO8lsyw0jnEsRFrzs/wK6VqqM=";
+    fetchSubmodules = true;
+  };
+
+  # hass-addons contains a few patches
+  hass-addons = fetchFromGitHub {
+    owner = "home-assistant";
+    repo = "addons";
+    rev = "583a62a69fa92ca8fdf9a2f298270a50bb3663a1";
+    hash = "sha256-u+lvU7mlJZqCCDIT4n7WnHyzAd0nZCh9Ddvqjs4SkHA=";
+  };
+
+  frontendModules = buildNpmPackage {
+    pname = "${pname}-frontend";
+    inherit version;
+    src = "${src}/src/web/web-service/frontend";
+    npmDepsHash = "sha256-7UVfPICyIbHEClpr3p7eDR46OUzS8mVf6P7phnDpVLk=";
+    dontNpmBuild = true;
+  };
+in
+stdenv.mkDerivation {
+
+  inherit pname version src;
+
+  # warning _FORTIFY_SOURCE requires compiling with optimization (-O)
+  env.NIX_CFLAGS_COMPILE = "-O";
+
+  patches = [
+    ./dont-install-systemd-units.patch
```

Comment:
Why not?

Reply:
It would try to install them in their final location which fails in the sandbox

Reply:
Couldn't you modify the location to be inside the derivation?

Reply:
It would try to install them in their final location (not respecting the cmake install path), which fails in the sandbox

Reply:
> Couldn't you modify the location to be inside the derivation?

What for though? The units themselves are part of the nixos module.

Reply:
> What for though? The units themselves are part of the nixos module.

I could have sworn there was a standard path for systemd units. Oh well, feel free to resolve this.